exporters/trace/build.gradle

```diff
@@ -4,4 +4,5 @@ dependencies {
     api(libraries.opentelemetry_api)
     api(libraries.opentelemetry_sdk)
     api(libraries.google_cloud_trace)
+    testImplementation('junit:junit:4.13')
```

Comment:
Sorry, missed this: Could you please define a constant in https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/5ecabaab3b959a8e882bd2333691137e49d35bc5/build.gradle#L20 and refer to it here?
This will help us use the same version across the repository.

e.g. `testLibraries.junit`

Reply:
Changed and thanks for the advice. I'm still learning about how gradle works and this seems much cleaner :)